test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -154,21 +156,70 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/hello-world-servic
 puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service ; comment"
           service-maps      (->> bootstrap-config
                                  (StringReader.)
-                                 parse-bootstrap-config!
+                                 vector
+                                 parse-bootstrap-configs!
                                  (map service-map))]
       (is (= (count service-maps) 2))
       (is (contains? (first service-maps) :HelloWorldService))
       (is (contains? (second service-maps) :TestService)))))
 
+(deftest multiple-bootstrap-files
+  (testing "Multiple bootstrap files can be specified directly on the command line"
+    (with-test-logging
+      (let [bootstrap-one "./dev-resources/bootstrapping/cli/split_bootstraps/one/bootstrap_one.cfg"
+            bootstrap-two "./dev-resources/bootstrapping/cli/split_bootstraps/two/bootstrap_two.cfg"
+            bootstrap-path (format "%s,%s" bootstrap-one bootstrap-two)
+            app (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-path])
+            test-svc (get-service app :TestService)
+            hello-world-svc (get-service app :HelloWorldService)]
+        (is (logged?
+              (format "Loading bootstrap configs:\n%s\n%s"
+                      (fs/absolute bootstrap-one)
+                      (fs/absolute bootstrap-two))
+              :debug))
+        (is (= (test-fn test-svc) :cli))
+        (is (= (hello-world hello-world-svc) "hello world")))))
+  (testing "A path containing multiple .cfg files can be specified on the command line"
+    (with-test-logging
+      (let [bootstrap-path "./dev-resources/bootstrapping/cli/split_bootstraps/both/"
+            app (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-path])
+            test-svc (get-service app :TestService)
+            hello-world-svc (get-service app :HelloWorldService)]
+        (is (logged?
+              ; We can't know what order it will find the files on disk, so just
+              ; look for a partial match with the path we gave TK.
+              (re-pattern (format "Loading bootstrap configs:\n%s"
+                                  (fs/absolute bootstrap-path)))
+              :debug))
+        (is (= (test-fn test-svc) :cli))
+        (is (= (hello-world hello-world-svc) "hello world")))))
+  (testing "A path containing both a file and a folder can be specified on the command line"
```

Comment:
Ah, yep, those are different services.  Mixed up my booostraps.  :smile: 

My recollection was that the current implementation would just load a single instance of the service even if the same namespace were to appear multiple times in the file.  I tried doing a split bootstrap install where I duplicated the namespace in each and it seemed to work fine with changes on your branch.  Assuming you agree it does / should as well, I was just thinking you could add a test onto this PR which demonstrates that.  If it turns out something is broken and you think it would be better to fix that as part of the work to disallow different service namespaces that implement the same protocol in [TK-211](https://tickets.puppetlabs.com/browse/TK-211), I'm fine deferring out the addition of a duplicates test in this PR.  Make sense?